python/FloodSAR/flood_sar_udf.ipynb

```diff
@@ -0,0 +1,224 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "5d9dbb2a",
+   "metadata": {},
+   "source": [
+    "# Flood extent using Sentinel 1\n",
+    "\n",
+    "Flood extent can be determined using a change detection approach on Sentinel-1 data. In this notebook we have tried in adopting (UN SPIDER's)[https://www.un-spider.org/advisory-support/recommended-practices/recommended-practice-google-earth-engine-flood-mapping] recommendation practice for computing flood extent by implementing openeo process."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "990b2125",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Authenticated using refresh token.\n"
+     ]
+    }
+   ],
+   "source": [
+    "# import necessary packages\n",
+    "import openeo\n",
+    "from openeo.api.process import Parameter\n",
+    "import json\n",
+    "from pathlib import Path\n",
+    "import matplotlib.pyplot as plt\n",
+    "import rasterio\n",
+    "import numpy as np\n",
+    "\n",
+    "# connect with the backend\n",
+    "eoconn = openeo.connect(\"openeo.vito.be\").authenticate_oidc()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "894a7ad2",
+   "metadata": {},
+   "source": [
+    "User can choose among different backend available [here](https://hub.openeo.org/) to connect to the backend. Rrgarding the authentication process OpenID connect (oidc) is recommended, but not always straightforward to use. In cases where you are unable to connect with the backend use basic authentication method explained [here](https://openeo.org/documentation/1.0/authentication.html#openid-connect)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "a23f66b3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# function to load geojson file\n",
+    "def read_json(path: Path) -> dict:\n",
+    "    with open(path) as input:\n",
+    "        field = json.load(input)\n",
+    "        input.close()\n",
+    "    return field"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "74cbac83",
+   "metadata": {},
+   "source": [
+    "To use the data collection, a user must use the correct backend with the data collection. Then using load_collection, they can specify bands, temporal extent (i.e. interested time interval) and even spatial extent. In this example, we have loaded Sentinel1 data from Terrascope available in VITO backend over a specified spatial extent."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "4e33a86a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# load before flash flood params\n",
+    "before_date = [\"2021-05-12\",\"2021-05-12\"]\n",
+    "after_date = [\"2021-06-18\", \"2021-06-18\"]\n",
+    "spatial_param = read_json(\"../aoi/cologne_all.geojson\")\n"
```

Comment:
After moving to separate folders per use-case the paths in the Jupyter Notebooks are not correct anymore, right?

Reply:
updated